code/modules/clothing/head/helmet.dm

```diff
@@ -436,8 +435,11 @@
 	visible_message("<span class='warning'>[src] powers up!</span>")
 	playsound(src, 'sound/machines/ping.ogg', 30, TRUE)
 	RegisterSignal(magnification, COMSIG_SPECIES_LOSS, .proc/make_fall_off)
+	INVOKE_ASYNC(src, /obj/item/clothing/head/helmet/monkey_sentience.proc/connect, user)
```

Comment:
You don't need to type out full path if  it's the same as the type the call is in. `.proc/connect` should be enough here.